upup/models/cloudup/resources/addons/spotinst-kubernetes-cluster-controller.addons.k8s.io/v1.8.0.yaml.template

```diff
@@ -97,7 +97,7 @@ spec:
     spec:
       containers:
       - name: spotinst-kubernetes-cluster-controller
-        image: spotinst/kubernetes-cluster-controller:1.0.16
+        image: spotinst/kubernetes-cluster-controller:1.0.18
```

Comment:
When we update this, we also need to update this: https://github.com/kubernetes/kops/blob/master/upup/pkg/fi/cloudup/bootstrapchannelbuilder.go#L450

(I hope we're going to make all this much better with bundles real-soon-now, but for now that's what we need...)

Also, I recommend making the version in bootstrap channel builder `1.0.18-kops.1`, because in the past we've had a problem where we wanted to update the manifest, but the image itself didn't change versions.  The `kops.1` suffix lets you do that, but otherwise there's nothing we can do...